scheduler/src/cook/mesos/api.clj

```diff
@@ -1119,6 +1130,21 @@
       (-> instance
           (assoc :job (select-keys job [:uuid :name :status :state :user]))))))
 
+(defn cook-info-handler
+  "Handler for the /info endpoint"
+  [settings]
+  (let [auth-middleware (:authorization-middleware settings)
+        auth-scheme (str (or (-> auth-middleware meta :json-value) auth-middleware))
+        ;; We store the start-up time (ISO-8601) for reporting on the /info endpoint
+        start-up-time (tf/unparse iso-8601-format (t/now))]
+    (base-cook-handler
+      {:allowed-methods [:get]
+       :handle-ok (fn get-info-handler [_]
+                    {:authorization-scheme auth-scheme
```

Comment:
Should this be `:authentication-scheme`?

Reply:
Yes, that's a better name. Fixed.